fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -49,8 +50,9 @@ struct TenantManagementWorkload : TestWorkload {
 	const TenantName tenantNamePrefix = "tenant_management_workload_"_sr;
 	TenantName localTenantNamePrefix;
 
-	const Key specialKeysTenantMapPrefix = TenantMapRangeImpl::submoduleRange.begin.withPrefix(
-	    SpecialKeySpace::getModuleRange(SpecialKeySpace::MODULE::MANAGEMENT).begin);
+	const Key specialKeysTenantMapPrefix =
+	    TenantRangeImpl<true>::mapSubRange.begin.withPrefix(TenantRangeImpl<true>::submoduleRange.begin.withPrefix(
```

Comment:
This is a little hard to parse. Maybe consider reversing it using `withSuffix` instead so it can just be read in order